### PR TITLE
Fix for conda-forge/python_abi-feedstock#2

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -14,14 +14,14 @@
...
-    #"linux-ppc64le",
-    #"osx-64",
-    #"win-32",
-    #"win-64",
+    "linux-armv7l",
+    "linux-aarch64",
+    "linux-ppc64le",
+    "osx-64",
+    "win-32",
+    "win-64",
```
Comment (beckermr) [inline recipe/gen_patch_json.py]: uhhhh - were we not shipping repo data patches?!

Comment (isuruf) [reply recipe/gen_patch_json.py]: Yes, my mistake on a previous PR.

**Comment by isuruf** [comment]
Tests pass. Ready for a review.

**Comment by isuruf** [comment]
Thanks.

**Pull request merged**